src/js/components/Button/Button.js

```diff
@@ -276,7 +276,7 @@ const Button = forwardRef(
           direction="row"
           align="center"
           justify={justify || (align === 'center' ? 'center' : 'between')}
-          gap={gap}
+          gap={theme.button.gap || gap}
```

Comment:
```suggestion
          gap={gap || theme.button.gap}
```
should it be like this? theme would be fallback if gap prop is undefined